shared/helpers/view.js

```diff
@@ -14,6 +14,16 @@ module.exports = function (Handlebars) {
     viewOptions = options.hash || {};
     app = getProperty('_app', this, options);
 
+    _.forEach(_.omit(viewOptions, 'templateName'), function(value, key) {
+      if (
+        key.match(/[A-Z]/g) &&
+        !app.modelUtils.isModel(value) &&
+        !app.modelUtils.isCollection(value)
+      ) {
+        throw new Error('Template properties should be passed in snake_case: "' + key + '"')
```

Comment:
Hm, are you suggesting fixing it so that mixed-case properties work? or should I change the error message to indicate that properties should always be lowercase?